model/error.yaml

```diff
@@ -0,0 +1,31 @@
+groups:
+  - id: error
+    type: attribute_group
+    prefix: error
+    brief: >
+      This document defines the shared attributes used to
+      report an error.
+    attributes:
+      - id: id
+        brief: 'Describes a class of error operation has ended with.'
+        type:
+          allow_custom_values: true
+          members:
+            - id: other
+              value: "_OTHER"
+              brief: 'Any error instrumentation does not define custom value for.'
+        examples: ['timeout', 'java.net.UnknownHostException', 'server_certificate_invalid', 'Internal Server Error']
+        note: |
+          The `error.id` SHOULD be predictable and SHOULD have low cardinality.
+          Instrumentations SHOULD document the list of errors they report.
+
+          The cardinality of `error.id` within one instrumentation library SHOULD be low, but
+          telemetry consumers that aggregate data from multiple instrumentation libraries and applications
+          should be prepared for `error.id` to have high cardinality at query time, when no
+          additional filters are applied.
```

Comment:
Do we want to expand here and say that usually, such cardinality concerns can be addressed by combining `error.id` with `service.name`?

Reply:
it doesn't feel needed to me, but could be easily persuaded

Reply:
I think it's a combination of attributes: `service.name` and `service.address`, but maybe other ones (depending on the convention, application type, etc), so I feel it'd be hard to provide anything concrete.  

I recall some time ago we discussed non-normative docs, this could be a great subject in "how to read HTTP semantic conventions when you implementing an observability backend".